include/staff/templates/merge-tickets.tmpl.php

```diff
@@ -97,6 +97,26 @@
     </label>
 </div>
 <br/><br/>
+<div id="child-status">
+&nbsp;&nbsp;&nbsp;
+    <label class="inline checkbox">
+        <?php echo __('Child Status');?>
+        <select id="statusId" name="statusId">
+        <?php
+        $states = array('closed');
+        foreach (TicketStatusList::getStatuses(
+                    array('states' => $states)) as $s) {
+            if (!$s->isEnabled()) continue;
+            echo sprintf('<option value="%d">%s</option>',
+                    $s->getId(),
+                    __($s->getName()));
```

Comment:
I believe this need to be `$s->getLocalName()` for translated status name.

Reply:
It looks like it's good since it has __() around it. Not sure why, but when I dump just getLocalName(), it's not actually translated.

Reply:
That's not how inline translation works - I'll explain in person when you have a second.

Reply:
Kevin just explained it to me. Didn't realize those were supposed to be manually edited.